### ISSUE TITLE
🐛 PDF.js doesn't load

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/adventist_knapsack#731]
# Summary

related: 
- [x] #726 

PDF.js isn't loading. 

The following was imported via oai bulkrax. 

![Image](https://github.com/user-attachments/assets/31d51d7e-1ff4-41eb-a86b-9165c7a379e3)

![Image](https://github.com/user-attachments/assets/a7c1cee9-b6b6-4d00-940f-4311a8407e5c)

# Acceptance Criteria

- [ ] PDF.js should render PDFs

# Testing Instructions

Feature flip to PDF.js.
Via bulkrax or manually, import a PDF file. 
check the pdf.js viewer on the work's show page.

# Notes

**Comment by sjproctor** [comment]
This is what I am getting when adding a PDF to a generic work.


![Image](https://github.com/user-attachments/assets/9b5bcd6c-8520-41e4-bb6d-3083f5e6a82a)



![Image](https://github.com/user-attachments/assets/66d8b3c1-0f9f-47b9-92a4-5c4c05b8ddae)

**Comment by kirkkwang** [comment]
Hey @sjproctor the file set looks like a jpg, just wanted to double that if this was a actually a PDF or not.

**Comment by ShanaLMoore** [comment]
# QA RESULTS: Pass ✅ 

When PDF.js is enabled and we attach a PDF file to a [work](https://demo.s2.adventistdigitallibrary.org/concern/theses/4d896a48-c0c7-42fe-8618-d8e1fda40ff4?locale=en):

![image](https://github.com/user-attachments/assets/e4c063b3-bab2-4e77-81bc-8a6c048df684)

**Comment by KatharineV** [comment]
@laritakr just confirming, this ticket doesn't test the logic you were discussing the other day? This ticket is only about PDF.js behavior when the feature is flipped, not about testing PDF.js as the preliminary viewer before the UV kicks in?

**Comment by laritakr** [comment]
> this ticket doesn't test the logic you were discussing the other day?

Correct. I am finishing up the work today to use PDF.js and UV as we discussed.

**Comment by KatharineV** [comment]
Works as expected on staging. Created this new work: https://adl.s2.adventistdigitallibrary.org/concern/conference_items/67eff180-3e9b-4f42-8192-363fefef841f?locale=en 

Immediately after upload: ![Image](https://github.com/user-attachments/assets/5480af5c-ff4c-409e-90c4-59ac511471c4)

After refresh: ![Image](https://github.com/user-attachments/assets/c4917f18-d539-42d1-8587-dda520049d55)

Flipped the feature back to UV and PDF.js was still present and working great with this work.

**Comment by KatharineV** [comment]
Resolved. All my testing and all the new works on prod lately are showing in PDF.js. I have not seen any errors.
